modules/seqexec/server/src/main/resources/Niri.xml

```diff
@@ -0,0 +1,297 @@
+        <command name="niri::stop">
+            <record>dc:stop</record>
+            <description>Stop observation</description>
+        </command>
+    </Apply>
+    <Apply name="niri::apply">
+        <top>niri</top>
+        <apply>apply</apply>
+        <car>applyC</car>
+        <description>NIS Primary Apply Record </description>
+        <command name="niri::endObserve">
+            <record>endObserve</record>
+            <description>Notify end of observation</description>
+        </command>
+        <command name="niri:config">
+            <record>cc:covSel</record>
+            <description>Window Cover control</description>
+            <parameter name = "windowCover">
+                <channel>cc:covSel.A</channel>
+                <type>STRING</type>
+            </parameter>
+        </command>
+    </Apply>
+    <Apply name="nis::apply">
+        <top>nis</top>
+        <apply>apply</apply>
+        <car>applyC</car>
+        <description>NIS Primary Apply Record </description>
+        <command name="nis::config">
+            <description>Setup a Configuration</description>
+            <!--<parameter name="disperser">-->
+                <!--<channel>grism:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>grism</description>-->
+            <!--</parameter>-->
+            <!--<parameter name="readMode">-->
+                <!--<channel>readmode:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>readmode</description>-->
+            <!--</parameter>-->
+            <parameter name="numCoAdds">
+                <channel>dcSetup:cad.H</channel>
+                <type>INTEGER</type>
+                <description>Coadds</description>
+            </parameter>
+            <!--<parameter name="mask">-->
+                <!--<channel>fpmask:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>fpmask</description>-->
+            <!--</parameter>-->
+            <!--<parameter name="camera">-->
+                <!--<channel>camera:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>camera</description>-->
+            <!--</parameter>-->
+            <!--<parameter name="beamSplitter">-->
+                <!--<channel>beamsplit:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>beamsplitter</description>-->
+            <!--</parameter>-->
+            <parameter name="exposureTime">
+                <channel>dcSetup:cad.F</channel>
+                <type>DOUBLE</type>
+                <description>Integration time for IS</description>
+            </parameter>
+            <!--<parameter name="builtinROI">-->
+                <!--<channel>roi:menu</channel>-->
+                <!--<type>STRING</type>-->
+                <!--<description>Builtin ROI</description>-->
+            <!--</parameter>-->
```

Comment:
Why are so many of the parameters commented out, out of curiosity?

Reply:
The file is created by importing the old Seqexec configuration files. I have to take out the Enum channels because the XML format does not have support them (in the old Seqexec, all of them are read as strings). Those channels are created in the code, but I left them in the XML because of their info.